### ISSUE TITLE
Restoring Nuget packages fails when using the build info options

### ISSUE THREAD

**Issue by ThomasPiskol** [jfrog/jfrog-cli-go#244]
Consider the following scenario:

```
RootFolder
   RootSolution.csproj
   RootSolution.sln
   packages.config
   ...
   SubFolder
      LibWithNoDependency.sln
      LibWithNoDependency.csproj
```

The following command will fail:
```
jfrog rt nuget restore my-nuget-repo --solution-root=C:\RootFolder --build-name=my-build --build-number=1
```

With this message:
```
[Error] Unsupported project dependencies for project: LibWithNoDependency
```

This command will not fail surprisingly, if the build info parameters are omitted:
```
jfrog rt nuget restore my-nuget-repo --solution-root=C:\RootFolder
```

or if there's an empty package.config file present in **SubFolder**:

Empty file:
```xml
<?xml version="1.0" encoding="utf-8"?>
<packages>
</packages>
```
... placed in **RootFolder\SubFolder**:

```
RootFolder
   RootSolution.csproj
   RootSolution.sln
   packages.config
   ...
   SubFolder
      LibWithNoDependency.sln
      LibWithNoDependency.csproj
      packages.config
```

Any project without a dependency should not cause an error.

JFrog CLI version:
```
jfrog -v
jfrog version 1.20.2
```

Sample projects for this issue:
[JFrogCLIBug.zip](https://github.com/jfrog/jfrog-cli-go/files/2515687/JFrogCLIBug.zip)

**Comment by AlexeiVainshtein** [comment]
Hi @ThomasPiskol 

I wanted to update you that we have released a new version of the JFrog CLI (1.21.0) that includes a fix for this issue. This issue occurred since the previous versions of the CLI expected to have a package.json or assets file. If those files were not presented then we were failing the collection of the build info. With the new release we are not failing but just writing to the log the information. 
We will appreciate your feedback with the new version and if this fixed your issue.

Thanks,

**Comment by ThomasPiskol** [comment]
Thanks for the fix, now it's working as expected.
